src/lib/tzcode/localtime.c

```diff
@@ -1664,12 +1666,11 @@ timesub(const time_t *timep, int_fast32_t offset, const struct state *sp,
 		idays -= ip[tmp->tm_mon];
 	tmp->tm_mday = idays + 1;
 	tmp->tm_isdst = 0;
-#ifdef TM_GMTOFF
-	tmp->TM_GMTOFF = offset;
-#endif /* defined TM_GMTOFF */
+	tmp->tm_gmtoff = offset;
 	return tmp;
 }
 
+#if 0
```

Comment:
Dead code, right?
Commit "datetime: use tzdata aka Olson for timezone resolution".

Reply:
There is no such. 🤷‍♂️
```
11:51 $ git log -9 --oneline
832b2725e (HEAD -> tsafin/gh-6751-olson-2) datetime: Olson timezones changelog and doc request
c756bf08a datetime: use lookaside pointer for tzalloc
c7f60f2b7 datetime: interval initialization via pairs
20cd172ad datetime: datetime.TZ array
ad70bb3bd datetime: implement date.isdst
04427485b datetime: use Olson for timezone name
6e777cfde datetime: localtime.c switched to Tarantool struct tnt_tm
b3c87cce5 datetime: primary and aliases zones in Olson
21abf14bc datetime, build - tzcode/olson integrated into cmake
```

Reply:
But well, I see